web/src/connectors/edit-job-connector/edit-job-connector.tsx

```diff
@@ -98,6 +98,11 @@ const EditJobConnector: FC<EditJobConnectorProps> = ({
             }
         };
 
+        metadata.props['owners'] = {
+            validationMessage: 'You should select at least 1 owners are required',
```

Comment:
Should this be like for other messages: "For "Owners" at least 1 owner is required"?

Reply:
Changed

Reply:
Optional: Since now most of the words in quotes maybe, it makes sense to use them everywhere, what do you think?